.github/dependabot.yml

```diff
@@ -13,6 +13,7 @@ updates:
       # Setting a timezone so we let dependabot worry about BST
       timezone: 'Europe/London'
     versioning-strategy: increase
+    open-pull-requests-limit: 10
```

Comment:
Super minor thing (not blocking) is we've ordered these keys A-Z

(Except for `package-ecosystem` which is used as a title-ish)